app/models/pglogical_subscription.rb

```diff
@@ -76,20 +78,11 @@ def self.delete_all(list = nil)
   end
 
   def disable
-    MiqPglogical.with_connection_error_handling { pglogical.disable_subscription(id).check }
+    self.class.with_connection_error_handling { pglogical.disable_subscription(id).check }
   end
 
   def enable
-    MiqPglogical.with_connection_error_handling { pglogical.enable_subscription(id).check }
-  end
-
-  def self.pglogical(refresh = false)
-    @pglogical = nil if refresh
-    @pglogical ||= PG::LogicalReplication::Client.new(connection.raw_connection)
-  end
-
-  def pglogical(refresh = false)
```

Comment:
These methods are now inherited via `MiqPglogical::ConnectionHandling`, since it is required that the `with_connection_error_handling` method have access to this method.  Probably doesn't require including the instance method, since I think this class is the only one that has it, but included it in the new module just to keep it all in the same place.